docs/intelligentapps/finetune.md

```diff
@@ -0,0 +1,278 @@
+---
+Order: 5
+Area: intelligentapps
+TOCTitle: Finetune
+ContentId:
+PageTitle: Finetune AI Models
+DateApproved:
+MetaDescription: Use custom dataset to finetune a generative AI model in the Azure cloud or locally with GPUs. Deploy the finetuned model to the Azure cloud or download adapter file.
+MetaSocialImage:
+---
+
+# Finetune models
+
+# **[Preview]** Introducing Remote Development with AI Toolkit for VS Code
+
+In addition to local development, the AI Toolkit for VS Code now supports remote development. This feature enables users to provision for Azure Container Apps to run model fine-tuning and inference endpoint in the cloud.
+
+## Prerequisites
+1. To run the model fine-tuning and inference in your remote Azure Container Apps Environment, make sure your subscription has enough GPU capacity. Submit a [support ticket](https://azure.microsoft.com/support/create-ticket/) to request the required capacity for your application. [Get More Info about GPU capacity](https://learn.microsoft.com/en-us/azure/container-apps/workload-profiles-overview)
```

Comment:
These are a lot of steps to put under a prerequisites section. Can these steps be a "Set up your project/environment" section?

Reply:
done